### DISCUSSION TITLE
Eleven Clock replaces T-Clock

### DISCUSSION THREAD

**Discussion by Alinghy17** [martinet101/ElevenClock#1286] [General]
Hello everyone, 
I'm new to this valuable section: I wanted to suggest, if I may, an implementation for Eleven Clock. I remember perfectly that in T-Clock there was an interesting feature, the day of the week, positioned before the date: if I'm not mistaken, I don't think there is this possibility of choice in the Eleven Clock options. Thank you for anything you can tell me about it. Have a good day

**Comment by marticliment** [comment]
Hello,

In ElevenClock you can specify custom date and time formats.

**Reply by Alinghy17** [reply]
Sorry, 
but this doesn't answer my question/suggestion. 
It is well clear that one can set the display of the time and date, while I asked if there is, as there was in T -Clock, the possibility of also setting the display of the current day of the week, given that can already set the display of the number of  current week, in case I had missed this option relating to the display of the day, in the various setup options. 
So, looking at you answer, I understand that this option is not available, I wonder if it ever will be..., it could be useful. 
Thanks and have a good afternoon.
A.

**Reply by marticliment** [reply]
As said, you can enable the option to specify custom date and time formats, feasturing weekday, week number, offsets, and even custom text

On [this website](https://strftime.org/) you can find all the different elements that you can add to your current time.

![image](https://github.com/marticliment/ElevenClock/assets/53119851/9fcccc19-745f-4dd8-93f8-db339fa6f55b)

**Reply by Alinghy17** [reply]
![Eleven Clock  - 2023-11-26](https://github.com/marticliment/ElevenClock/assets/152058125/e16b01fe-85a5-4d23-8d67-120f29cc3df9)
Thanks for the further reply,
since I'm not a skilled user, could you post the command to make the day of the week appear in the second line, under the time (%#H:%M:%S), ( ........ .) followed by the date (%d/%m/%y) ? I don't know how to write the command for the current day...
Thank you in advance.
Good afternoon
TO.

**Reply by marticliment** [reply]
Option 1:
```
%#H:%M:%S
%a %d/%m/%y
```

![image](https://github.com/marticliment/ElevenClock/assets/53119851/77f43345-c047-4301-963c-b402f6e4658d)

Option 2:
```
%#H:%M:%S
%A %d/%m/%y
```
![image](https://github.com/marticliment/ElevenClock/assets/53119851/ec34e2c7-eb44-4583-a923-e2d998f538e7)

**Reply by Alinghy17** [reply]
Perfect, exactly what I want. 
Really appreciated the help given to me.
Good afternoon again.
Regards
A.
